typos.toml

```diff
@@ -7,32 +7,27 @@ extend-exclude = [
 ]
 ignore-hidden = false
 
-# Corrections take the form of a key/value pair. The key is the incorrect word
-# and the value is the correct word. If the key and value are the same, the
-# word is treated as always correct. If the value is an empty string, the word
-# is treated as always incorrect.
-
-# Match Whole Word - Case Sensitive
-[default.extend-identifiers]
-iy = "iy"       # Variable name used in bevy_gizmos. Probably stands for "y-axis index", as it's being used in loops.
-ser = "ser"     # ron::ser - Serializer
-SME = "SME"     # Subject Matter Expert
-Sur = "Sur"     # macOS Big Sur - South
-Masia = "Masia" # The surname of one of the authors of SMAA
-Ba = "Ba"       # Bitangent for Anisotropy
-ba = "ba"       # Part of an accessor in WGSL - color.ba
-
 # Match Inside a Word - Case Insensitive
 [default.extend-words]
 LOD = "LOD" # Level of detail
 TOI = "TOI" # Time of impact
 
 [default]
 locale = "en-us"
+# Ignored typos regexes
 extend-ignore-identifiers-re = [
```

Comment:
I believe it is easier to have everything in the same place. If I understand correctly, `default.extend-identifiers` is better used to define replacements or deny some usages.